showmore_dyn/showmore_dyn.js

```diff
@@ -0,0 +1,81 @@
+$(document).ready(function(){
+        handleNewWallItemBodies();
+
+        var mutationObserver = new MutationObserver(function(mutations) {
+                handleNewWallItemBodies();
+        });
+        mutationObserver.observe($("#content")[0], { attributes: false, characterData: false, childList: true, subtree: true, attributeOldValue: false, characterDataOldValue: false });
+});
+
+function handleNewWallItemBodies() {
+        $('.wall-item-body:not(.showmore-done)').each(function(i, el) {
```

Comment:
If you aren't using `i`, then you probably don't need `el` either. You can just use `$(this)` inside of the `$.each()`.